packages/components/src/styles/basiq/theme.css

```diff
@@ -0,0 +1,194 @@
+   * to ensure contrast with the foreground color */
+  --highlight-background: var(--basiq-slate-600); /* #6f46ed purple-300 */
+  --highlight-background-pressed: var(
+    --basiq-slate-800
+  ); /* #522acd purple-200 */
+  --highlight-background-invalid: var(
+    --quanta-candy
+  ); /* red-300 from dark theme */
+  --highlight-background-hover: var(--basiq-slate-400); /* #6f46ed purple-300 */
+  --highlight-foreground: var(--basiq-slate-50); /* white */
+  --highlight-foreground-pressed: var(--basiq-slate-200); /* #ddd */
+  --highlight-overlay: var(
+    --basiq-slate-400
+  ); /* rgb(from #6f46ed r g b / 15%) */
+  --color-invalid: var(--red-400); /* Renamed */
+  --color-pressed-invalid: var(--red-500); /* Renamed */
+}
+
+/* Layout properties */
+:root {
+  --layout-container-width: 1440px;
+  --default-container-width: 940px;
+  --narrow-container-width: 620px;
+}
+
+/* Windows high contrast mode overrides */
+@media (forced-colors: active) {
+  :root {
+    --background-color: Canvas;
+    --focus-ring-color: Highlight;
+    --text-color: ButtonText;
+    --text-color-base: ButtonText;
+    --text-color-hover: ButtonText;
+    --text-color-disabled: GrayText;
+    --text-color-placeholder: ButtonText;
+    --link-color: LinkText;
+    --link-color-secondary: LinkText;
+    --link-color-pressed: LinkText;
+    --border-color: ButtonBorder;
+    --border-color-hover: ButtonBorder;
+    --border-color-pressed: ButtonBorder;
+    --border-color-disabled: GrayText;
+    --field-background: Field;
+    --field-text-color: FieldText;
+    --overlay-background: Canvas;
+    --button-background: ButtonFace;
+    --button-background-pressed: ButtonFace;
+    --highlight-background: Highlight;
+    --highlight-background-pressed: Highlight;
+    --highlight-background-invalid: LinkText;
+    --highlight-foreground: HighlightText;
+    --highlight-foreground-pressed: HighlightText;
+    --color-invalid: LinkText;
+    --color-pressed-invalid: LinkText;
+  }
+}
+
+/* Dark: https://leonardocolor.io/theme.html?name=Dark&config=%7B%22baseScale%22%3A%22Gray%22%2C%22colorScales%22%3A%5B%7B%22name%22%3A%22Gray%22%2C%22colorKeys%22%3A%5B%22%23000000%22%5D%2C%22colorspace%22%3A%22RGB%22%2C%22ratios%22%3A%5B%22-1.12%22%2C%221.45%22%2C%222.05%22%2C%223.02%22%2C%224.54%22%2C%227%22%2C%2210.86%22%5D%2C%22smooth%22%3Afalse%7D%2C%7B%22name%22%3A%22Purple%22%2C%22colorKeys%22%3A%5B%22%235e30eb%22%5D%2C%22colorspace%22%3A%22RGB%22%2C%22ratios%22%3A%5B%22-1.12%22%2C%221.45%22%2C%222.05%22%2C%223.02%22%2C%224.54%22%2C%227%22%2C%2210.86%22%5D%2C%22smooth%22%3Afalse%7D%2C%7B%22name%22%3A%22Red%22%2C%22colorKeys%22%3A%5B%22%23e32400%22%5D%2C%22colorspace%22%3A%22RGB%22%2C%22ratios%22%3A%5B%22-1.12%22%2C%221.45%22%2C%222.05%22%2C%223.02%22%2C%224.54%22%2C%227%22%2C%2210.86%22%5D%2C%22smooth%22%3Afalse%7D%5D%2C%22lightness%22%3A11%2C%22contrast%22%3A1%2C%22saturation%22%3A100%2C%22formula%22%3A%22wcag2%22%7D */
+@media (prefers-color-scheme: dark) {
```

Comment:
I have a personal opinion about this, that we should not include a dark mode theme by default. Most usecases for plone are public facing sites showing news items, events, blogs, products or whatever. These use cases usually don't have two separate themes or a theme switcher, it's something usually for dashboards, applications that are more focused on using some product or managing something.

Reply:
Let's remove it for now then, since for now it causes confusion.